### PR TITLE
Transalted to Bengali (BN) with some typo fixed

### PR DESCRIPTION
**Pull request by tfevan**

### CODE REVIEW

File: locales/bn/bn.json [hunk 1]
```diff
@@ -276,16 +276,16 @@
     "India": "ইন্ডিয়া",
     "Indonesia": "ফিজি",
     "Iran, Islamic Republic Of": "ফিনল্যাণ্ড",
-    "Iran, Islamic Republic of": "Iran, Islamic Republic of",
+    "Iran, Islamic Republic of": "ইরান (ইসলামি প্রজাতন্ত্র)",
```
Comment (caouecs) [inline locales/bn/bn.json:279]: Why two different translations for Iran ?

File: locales/bn/bn.json [hunk 2]
```diff
@@ -346,15 +346,15 @@
     "Mayotte": "মায়োট",
     "Mexico": "মেক্সিকো",
     "Micronesia, Federated States Of": "মাইক্রোনেশিয়া",
-    "Micronesia, Federated States of": "Micronesia, Federated States of",
+    "Micronesia, Federated States of": "মাইক্রোনেশিয়া, ফেডারেটেড স্টেট অফ",
```
Comment (caouecs) [inline locales/bn/bn.json:349]: Why two different translations ?

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/bn/bn.json (hunk 1 above)
Comment (tfevan) [reply locales/bn/bn.json:279]: The [main source](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L278) has duplicate key, I'll delete duplicate

File: locales/bn/bn.json (hunk 2 above)
Comment (tfevan) [reply locales/bn/bn.json:349]: The [main source](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L348) has duplicate key, I'll delete duplicate

File: locales/bn/bn.json (hunk 1 above)
Comment (caouecs) [reply locales/bn/bn.json:279]: No, my question is about the fact to have two different translations in Bengali for the same tag ; Of and of are normal.

File: locales/bn/validation.php [hunk 3]
```diff
@@ -11,9 +11,11 @@
 |
 */
 
+:attribute অবশ্যই গ্রহণ করতে হবে যখন :other সমান :value হয়
```
Comment (caouecs) [inline locales/bn/validation.php:14]: I think you can delete this line ;)

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/bn/bn.json (hunk 1 above)
Comment (tfevan) [reply locales/bn/bn.json:279]: I got a lot of incorrect bengali translation, most of country names are incorrect, like 

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L259) is `Honduras` but bengali translation is `England` (ইংল্যাণ্ড), 

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L280) is `Iraq` and bengali translation is `France` (ফ্রান্স). 

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L295) is `Kazakhstan` but bengali translation is `Greenland` (গ্রীনল্যাণ্ড),

[this](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L311) is `Labanon` but bengali translation is `Hong Kong` (হং কং)

I didn't touch this files, [this](https://github.com/Laravel-Lang/lang/pull/1928/commits/8141e75118361f5ded730bdcdb1675775e902473#diff-592c253b8b0a994772dcdfd84e643773275bde1021ff21d24210d87947f1c831L278) is one of those.

File: locales/bn/validation.php (hunk 3 above)
Comment (tfevan) [reply locales/bn/validation.php:14]: Are you sure I can delete this line ? `'accepted_if'          => 'এই ফিল্ডটি তখন গ্রগণ করা হবে যখন :other হবে :value',`

Comment (tfevan) [reply locales/bn/validation.php]: That was a silly mistake, my bad.

File: locales/bn/bn.json (hunk 1 above)
Comment (tfevan) [reply locales/bn/bn.json:279]: I'll send a new PR with all the country name

**Comment by caouecs** [comment]
Thank you

**Pull request merged**